src/client/v2/indexer/lookupAssetTransactions.ts

```diff
@@ -61,7 +61,7 @@ export default class LookupAssetTransactions extends JSONRequest {
    *        .do();
    * ```
    *
-   * @param type - one of `pay`, `keyreg`, `acfg`, `axfer`, `afrz`, `appl`
+   * @param type - one of `pay`, `keyreg`, `acfg`, `axfer`, `afrz`, `appl`, `stpf`
```

Comment:
I'm surprised by the need to support filtering asset related transactions by state proofs, but I don't see the big picture either.